ghedesigner/ground_loads.py

```diff
@@ -191,55 +190,39 @@ def process_two_day_loads(self) -> None:
         # the peak load of the month occurs on, and pulling a 48-hour load
         # profile -- the day before and the day of
 
-        hours_in_day = 24
         hours_in_year = len(self.hourly_rejection_loads)
 
         # Add the last day of the year to the beginning of the loads to account
         # for the possibility that a peak load occurs on the first day of the
         # year
 
-        hourly_rejection_loads = (self.hourly_rejection_loads[
-                                  hours_in_year - hours_in_day: hours_in_year] + self.hourly_rejection_loads)
-        hourly_extraction_loads = (self.hourly_extraction_loads[
-                                   hours_in_year - hours_in_day: hours_in_year] + self.hourly_extraction_loads)
+        hourly_rejection_loads = self.hourly_rejection_loads[hours_in_year - HRS_IN_DAY:] + \
+            self.hourly_rejection_loads
+        hourly_extraction_loads = self.hourly_extraction_loads[hours_in_year - HRS_IN_DAY:] + \
+            self.hourly_extraction_loads
```

Comment:
I don't think you need the end bound since it was just intended to capture the last day.